addons/mass_mailing/models/mailing_list.py

```diff
@@ -157,7 +157,9 @@ def action_send_mailing(self):
         action.update({
             'context': {
                 **self.env.context,
+                'default_model_id': self.env['ir.model']._get_id('mailing.list'),
                 'default_contact_list_ids': self.ids,
+                'default_mailing_type': 'mail',
```

Comment:
When possible, keep alphabetical ordering :) 
```suggestion
                'default_contact_list_ids': self.ids,
                'default_mailing_type': 'mail',
                'default_model_id': self.env['ir.model']._get_id('mailing.list'),
```